tests/unit/ExampleTest.php

```diff
@@ -0,0 +1,22 @@
+<?php 
+class ExampleTest extends \Codeception\Test\Unit
```

Comment:
ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)